graylog2-web-interface/src/components/users/UserCreate/FullNameFormGroup.jsx

```diff
@@ -19,12 +19,12 @@ import * as React from 'react';
 
 import { FormikFormGroup } from 'components/common';
 
-const FullNameFormGroup = () => (
-  <FormikFormGroup label="Full Name"
-                   name="full_name"
+const LastNameFormGroup = () => (
```

Comment:
Is this component still needed? The same folder has a new `LastNameFormGroup` component doing exactly the same.

Reply:
Thank you for pointing this out, this not needed anymore.